src/app/shared/header/case-header.component.ts

```diff
@@ -16,11 +17,14 @@ export class CaseHeaderComponent implements OnInit {
 
   ngOnInit(): void {
     this.caseTitle = new CaseField();
-    if (this.caseDetails.state.title_display) {
+    if (!this.isDraft() && this.caseDetails.state.title_display) {
       this.caseTitle.label = this.caseDetails.state.title_display;
       this.caseFields = this.getCaseFields();
     }
   }
+  isDraft(): boolean {
```

Comment:
This was needed as otherwise view draft wasn't working missing from previous draft story.